packages/lodestar/src/chain/chain.ts

```diff
@@ -372,4 +374,18 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
     return state;
   }
 
+  /**
+   * Wait until now >= genesisTime
+   */
+  private async waitForGenesisTime(genesisTime: number): Promise<void> {
+    let now = Date.now();
+    while (genesisTime - now > 0) {
+      // wait 15s or time to genesis, whichever is shorter
+      const diff = Math.min(genesisTime - now, 15000);
+      this.logger.info("Waiting for genesis", {waitTime: Math.floor(diff / 1000)});
```

Comment:
- maybe print it in minutes?
- add time unit to log

Reply:
oh noes wait, but it would just print 15 s all the time? I think log should display genesisTime-now not diff :smile: @wemeetagain

Reply:
XD yea, fixed